scripts/build_no.sh

```diff
@@ -23,12 +23,10 @@
 #
 #####################################################################
 
-# Fail on first error
 set -e
 
-GIT_ROOT=$(git rev-parse --show-toplevel)
 DEV_BUILD_NUMBER=9999
-BUILD_NUMBER_FILE="${GIT_ROOT}/BUILD_NUMBER"
+BUILD_NUMBER_FILE="BUILD_NUMBER"
```

Comment:
Wouldn't it make more sense to do something like:
```bash
if [[ -z "${BUILD_NUMBER_FILE}" ]]; then
  BUILD_NUMBER_FILE="${GIT_ROOT}/BUILD_NUMBER"
fi
```
And then provide the path of `BUILD_NUMBER_FILE` in Nix?
What do you think @PombeirP?

Reply:
I don't understand in which situation this script gets called and .git isn't there. Normally it is passed to Nix [here](https://github.com/status-im/status-react/blob/develop/ci/android.groovy#L38) where we have a Git repo. Unless it is happening [here](https://github.com/status-im/status-react/blob/develop/src/status_im/utils/build.clj#L25), but if that is the case, then I don't understand the need to add the `VERSION` to the jsbundle Nix expression, since it'll cause the bundle to be invalidated any time the `VERSION` file changes.

But yeah, I agree that passing an env var to the file in this situation is better as it still allows us having the script live anywhere and still have it behave correctly.